Userland/Applications/PixelPaint/Tools/BrushTool.cpp

```diff
@@ -79,6 +89,11 @@ Color BrushTool::color_for(GUI::MouseEvent const& event)
 
 void BrushTool::draw_point(Gfx::Bitmap& bitmap, Gfx::Color const& color, Gfx::IntPoint const& point)
 {
+<<<<<<< HEAD
+=======
+    constexpr auto flow_scale = 10;
+
+>>>>>>> 5cbbc44dbb (PixelPaint: Fixup improvements)
```

Comment:
Oops, this is a remnant of a merge conflict! I see you did remove the mess in the second commit, but please remove it from this one instead. Making sure everything builds after each commit makes it easier to `git bisect` later.

Reply:
Hey @AtkinsSJ, i removed my rebase artifacts... thanks for checking :-)